cellarium/cas/client.py

```diff
@@ -642,6 +794,69 @@ def search_10x_h5_file(
             feature_names_column_name=feature_names_column_name,
         )
 
+    def search_matrix(
+        self,
+        matrix: t.Union[str, "anndata.AnnData"],
+        chunk_size=CHUNK_SIZE_SEARCH_DEFAULT,
+        cas_model_name: str = "default",
```

Comment:
This should probably be
```suggestion
        cas_model_name: t.Union[str, bool] = True,
```
And that would protect against the (rare but possible) case where there is a model named "default"

Reply:
Can I maybe do cas_model_name: t.Optional[str] = None. If None then default model is used? 
Seems odd making it a bool, what should happen if someone passes False?

Reply:
Ah, that's a great point.  I like your way better :)

Reply:
So yeah, I like optional string